docs/tutorial/single-node.rst

```diff
@@ -13,7 +13,7 @@ three entire disks are required to be available on the machine.
 
 .. note::
 
-   Development is underway to allow for loopback device support.
+   Upstream Ceph development is underway to allow for loopback device support.
```

Comment:
The dev tasks referred to here need to be done in MicroCeph, from an upstream Ceph POV there is nothing to be done

Reply:
Okay, I was thrown by "this will filter down to MicroCeph". If the work is being done in MicroCeph, what is meant by "filter down"?

Reply:
Ah, good observation. I believe this was meant to refer to a dependency in snapd (which has by now been resolved though)